task03-one-piece-website/onepiece.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
```

Comment:
Let's fix the indentation on the HTML, which is not consistent throughout the file.